Userland/Applications/PixelPaint/Tools/LineTool.cpp

```diff
@@ -100,13 +100,11 @@ void LineTool::on_mousemove(Layer* layer, MouseEvent& event)
     } else {
         m_line_end_position = layer_event.position();
     }
-
     if (layer_event.alt()) {
         m_line_start_position = m_drag_start_position + (m_drag_start_position - m_line_end_position);
     } else {
         m_line_start_position = m_drag_start_position;
     }
-
```

Comment:
Like Linus said, could you remove these whitespace changes? (also the one above here that linus commented on)

Reply:
Not sure why, but I'm not seeing these on my end.  Are they still there?

Reply:
Yes, still there.
(In case you didn't notice this change is actually removing whitespace not adding it)
You can also just try copy pasting the original file from https://github.com/SerenityOS/serenity/blob/master/Userland/Applications/PixelPaint/Tools/LineTool.cpp on top of your local copy

Reply:
Okay I think both should be fixed now 👍